omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -0,0 +1,509 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2016-present Datadog, Inc.
+
+require './lib/ostools.rb'
+require 'json'
+
+name 'datadog-agent-integrations'
+
+license "BSD-3-Clause"
+license_file "./LICENSE"
+
+source git: 'https://github.com/DataDog/integrations-core.git'
+relative_path 'integrations-core'
+
+integrations_core_version = ENV['INTEGRATIONS_CORE_VERSION']
+if integrations_core_version.nil? || integrations_core_version.empty?
+  integrations_core_version = 'master'
+end
+
+default_version integrations_core_version
+
+dependency 'datadog-agent'
+
+if with_python_runtime? "2"
+  dependency 'datadog-agent-integrations-py2-dependencies'
+
+  whitelist_file "embedded/lib/python2.7/site-packages/.libsaerospike"
+  whitelist_file "embedded/lib/python2.7/site-packages/psycopg2"
+  whitelist_file "embedded/lib/python2.7/site-packages/wrapt"
+  whitelist_file "embedded/lib/python2.7/site-packages/pymqi"
+end
+
+if with_python_runtime? "3"
+  dependency 'datadog-agent-integrations-py3-dependencies'
+
+  whitelist_file "embedded/lib/python3.11/site-packages/.libsaerospike"
+  whitelist_file "embedded/lib/python3.11/site-packages/aerospike.libs"
+  whitelist_file "embedded/lib/python3.11/site-packages/psycopg2"
+  whitelist_file "embedded/lib/python3.11/site-packages/pymqi"
+end
+
+gcc_version = ENV['GCC_VERSION']
+if gcc_version.nil? || gcc_version.empty?
+  gcc_version = '10.4.0'
```

Comment:
`GCC_VERSION` is set in the docker images when we build GCC. If it isn't set, then only the distribution provided `gcc` (`4.x` IIRC) is available.

I'd expect that defaulting to `10.4.0` would fail (but isn't actually used)

Reply:
I'm not familiar with the context under which this was introduced, but interestingly, only the py3 definition had this reference to choosing the gcc_version, the py2 didn't.

It's only referenced here: https://github.com/DataDog/datadog-agent/blob/68887b6328fb842eed4daff2a139dce0e3104660/omnibus/config/software/datadog-agent-integrations.rb#L189-L193

Reply:
https://github.com/DataDog/datadog-agent/pull/17978

Reply:
Thanks @FlorentClarret for the context. Anyhow this just keeps what the py3 definition already had, so should there be anything to address here I think it would have to be separate work.

Reply:
Fine by me